CHANGELOG.md

```diff
@@ -1,15 +1,46 @@
 # Changelog
 
-## 0.3.0
+## 0.4.0
 
 ### Added
 
+- Support for `#[default = "..."]` in struct methods [[#436]](https://github.com/extendr/extendr/pull/436)
 - [**ndarray**] `TryFrom<&Robj>` for `ArrayView1<T>` and `ArrayView2<T>`, where `T` is `i32`, `f64`, `c64`, `Rint`, `Rfloat`, `Rcplx`, `Rstr`, `Rbool` [[#443]](https://github.com/extendr/extendr/pull/443)
-- `Sum` for scalars like `Rint`, `Rfloat` and `Rcplx`, which accept `Iterator<Item = &Rtype>` [[#454]](https://github.com/extendr/extendr/pull/454)
+- `Debug` trait implementation for `Rcplx` and `Complexes` [[#444]](https://github.com/extendr/extendr/pull/444)
 - `Nullable<T>` is now part of `extendr_api::prelude` [[#446]](https://github.com/extendr/extendr/pull/446)
 - `TryFrom<Robj>`, `From<Option<T>>`, `Into<Option<T>>` and their variations for `Nullable<T>` [[#446]](https://github.com/extendr/extendr/pull/446)
 - `Nullable<T>::map()` that acts on not null value and propagates `NULL` [[#446]](https://github.com/extendr/extendr/pull/446)
-- `Debug` trait implementation for `Rcplx` and `Complexes` [[#444]](https://github.com/extendr/extendr/pull/444)
+- [**ndarray**] Conversion from owned arrays (ie `ndarray::Array`) into `Robj` [[#450]](https://github.com/extendr/extendr/pull/450)
+- [**ndarray**] Documentation for the `robj_ndarray` module [[#450]](https://github.com/extendr/extendr/pull/450)
+- `Sum` for scalars like `Rint`, `Rfloat` and `Rcplx`, which accept `Iterator<Item = &Rtype>` [[#454]](https://github.com/extendr/extendr/pull/454)
+- Add a new `CollectRMatrix` trait that can be used to collect arbitrary iterables into an R matrix
+ [[#466]](https://github.com/extendr/extendr/pull/466)
+- Documentation for `RArray::new_matrix()` [[#466]](https://github.com/extendr/extendr/pull/466)
+
+### Changed
+
+- [**Documentation**] Use bindgen on docs.rs, to ensure newer R features will still be documented [[#426]](https://github.com/extendr/extendr/pull/426)
+- Unified how regular structs and ExternalPtr are treated (TODO: more detail needed) [[#433]](https://github.com/extendr/extendr/pull/433)
+- Updated `roxygen2` and `rextendr` versions used in integration tests [[#447]](https://github.com/extendr/extendr/pull/447)
```

Comment:
I think this integration test is mostly for CI. Do you run this on local often? (confession: I never, except for when the CI fails)

```suggestion
```

Reply:
No, I don't. Yes it's probably internal.